server/views/partials/prisonerBanner.njk

```diff
@@ -1,17 +1,23 @@
-<div class="app-prisoner-banner govuk-!-display-none-print">
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-one-quarter">
-      <p class="govuk-visually-hidden">{{ prisonerSummary.firstName }} {{ prisonerSummary.lastName | title }}'s details</p>
-      <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-6">Prison number: <span class="govuk-body" data-qa="prison-number">{{ prisonerSummary.prisonNumber }}</span></p>
-    </div>
-    <div class="govuk-grid-column-one-quarter">
-      <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-6">Entered this prison on: <span class="govuk-body" data-qa="reception-date">{{ prisonerSummary.receptionDate | formatDate('D MMM YYYY') }}</span></p>
-    </div>
-    <div class="govuk-grid-column-one-quarter">
-      <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-6">Release date: <span class="govuk-body" data-qa="release-date">{{ prisonerSummary.releaseDate | formatDate('D MMM YYYY') or 'N/A' }}</span></p>
-    </div>
-    <div class="govuk-grid-column-one-quarter">
-      <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-6">DOB: <span class="govuk-body" data-qa="dob">{{ prisonerSummary.dateOfBirth | formatDate('D MMM YYYY') }}</span></p>
-    </div>
-  </div>
+<div class="app-mini-profile-header govuk-!-margin-bottom-6">
```

Comment:
The previous prisoner banner had "no print" styles - do we need to do that on this new banner?
Also, if the requirement is that we **do** print the banner, does the anchor link need some styles or similar to control it's printing? (I cant remember what we do on other links?)

Reply:
Good spot - added the styles back.